tests/test_pyrflx.py

```diff
@@ -0,0 +1,291 @@
+import unittest
+
+from rflx.expression import UNDEFINED, Expr
+from rflx.model import Enumeration, ModularInteger, Number, Opaque, RangeInteger, Type
+from rflx.pyrflx.message import Field, Message
+from rflx.pyrflx.package import Package
+from rflx.pyrflx.pyrflx import PyRFLX
+from rflx.pyrflx.typevalue import (
+    EnumValue,
+    IntegerValue,
+    NotInitializedError,
+    OpaqueValue,
+    TypeValue,
+)
+
+
+# pylint: disable=too-many-public-methods
+class TestPyRFLX(unittest.TestCase):
+    def setUp(self) -> None:
+        self.testdir = "specs"
+        self.tlv: Message = PyRFLX([f"{self.testdir}/tlv_with_checksum.rflx"])["TLV"][
+            "Message"
```

Comment:
`setUp` is called for each test method. This makes the unittest really slow. A better solution is to use `setUpClass` and make a copy in `setUp`:

```
class TestPyRFLX(unittest.TestCase):
    testdir: str
    __tlv: Message

    @classmethod
    def setUpClass(cls) -> None:
        cls.testdir = "specs"
        cls.__tlv = PyRFLX([f"{cls.testdir}/tlv_with_checksum.rflx"])["TLV"][
            "Message"
        ]

    def setUp(self) -> None:
        self.tlv = copy(self.__tlv)

...

    def test_all_fields(self) -> None:
        self.assertEqual(self.tlv.fields, ["Tag", "Length", "Value", "Checksum"])
```

Reply:
Fixed by https://github.com/Componolit/RecordFlux/pull/158/commits/91889be32908f1f27383b00a59a97eca8d085b77.